src/antares/craft/model/study.py

```diff
@@ -169,20 +173,16 @@ def create_study_local(
     with open(desktop_ini_path, "w") as desktop_ini_file:
         desktop_ini_file.write(desktop_ini_content)
 
-    local_settings = StudySettingsLocal.model_validate(settings)
-    local_settings_file = IniFile(study_directory, InitializationFilesTypes.GENERAL)
-    local_settings_file.ini_dict = local_settings.model_dump(exclude_none=True, by_alias=True)
-    local_settings_file.write_ini_file()
-
     # Create various .ini files for the study
-    _create_correlation_ini_files(local_settings, study_directory)
+    _create_correlation_ini_files(study_directory)
```

Comment:
Just a high level organization comment:
we should move the implementation of local and api methods to the corresponding services modules.
Then we can import them here so that it does not change the API. Just the code organization will be more clear.